lib/Controller/ItemApiController.php

```diff
@@ -94,6 +94,13 @@ public function index(
                 );
                 break;
             default:
+                // Fallback in case people try getRead here
+                if ($getRead === false && $type === ListType::ALL_ITEMS) {
+                    $type = ListType::UNREAD;
+                } elseif ($getRead === false) {
+                    return ['message' => 'Setting getRead on an already filtered list is not allowed!'];
+                }
```

Comment:
What about a simpler (?) conditional branching @SMillerDev ? 🙂

```suggestion
                if ($getRead === false) {
                    if ($type !== ListType::ALL_ITEMS) {
                        return ['message' => 'Setting getRead on an already filtered list is not allowed!'];
                    }

                    $type = ListType::UNREAD;
                }
```

Reply:
I don't really think thats simpler

Reply:
I was about saving a redundant `$getRead === false` comparison 🤷